objecthash_test.py

```diff
@@ -83,6 +83,21 @@ def test_common(self):
         self.verify('["foo", "bar"]',
                     '32ae896c413cfdc79eec68be9139c86ded8b279238467c216cf2bec4d5f1e4a2')
 
+    def test_int(self):
+        self.verify('[123]',
+                    '1b93f704451e1a7a1b8c03626ffcd6dec0bc7ace947ff60d52e1b69b4658ccaa')
+        self.verify('[1, 2, 3]',
+                    '157bf16c70bd4c9673ffb5030552df0ee2c40282042ccdf6167850edc9044ab7')
+        self.verify('[123456789012345]',
+                    '3488b9bc37cce8223a032760a9d4ef488cdfebddd9e1af0b31fcd1d7006369a4')
+        self.verify('[123456789012345, 678901234567890]',
+                    '031ef1aaeccea3bced3a1c6237a4fc00ed4d629c9511922c5a3f4e5c128b0ae4')
+
+    def test_null(self):
+        # The same as common JSON
+        self.verify('[null]',
+                    '5fb858ed3ef4275e64c2d5c44b77534181f7722b7765288e76924ce2f9f7f7db')
+
```

Comment:
I've moved the [null] test out of .py to the test file.
I've reproduced the [int, int, ..]  tests for both common and 'python' json.